plugins/agents/dnsresponsetime/DnsResponseTime.go

```diff
@@ -0,0 +1,96 @@
+package dnsresponsetime
+
+import (
+	"strings"
+
+	"github.com/abrander/agento/plugins"
+	"github.com/abrander/agento/timeseries"
+	"github.com/miekg/dns"
+)
+
+type Data struct {
+	Loss   int    `json:"loss"`
+	Time   int64  `json:"time"`
+	Domain string `json:"domain"`
+	Server string `json:"server"`
+}
+
+type DnsResponseTime struct {
+	Data []Data `json:"data"`
+
+	Domains string `toml:"domain" json:"domain" description:"Domain(s) to query"`
```

Comment:
Maybe we should add that these can be separated by commas?

Reply:
Done